### DISCUSSION TITLE
changing right to left languages to left to right

### DISCUSSION THREAD

**Discussion by theBSH** [godotengine/godot-proposals#8440] [Editor]
as someone who has been on the internet for so long seeing the dockers change when I set the language to Persian is the worst thing ever there should at least be an option to turn that off its so bad for the workflow, and apps like blender have figured that out.

**Comment by theBSH** [comment]
![Screenshot 2023-11-16 042657](https://github.com/godotengine/godot-proposals/assets/118190096/e7b02f10-fad9-492a-b542-d1595cf70c41)
who can work like this ?

**Comment by Calinou** [comment]
Closing in favor of https://github.com/godotengine/godot/issues/84998.
